src/compiler/checker.ts

```diff
@@ -33000,10 +33027,14 @@ namespace ts {
             setNodeLinksForPrivateIdentifierScope(node);
         }
 
-        function checkAccessorDeclarationTypesIdentical(first: AccessorDeclaration, second: AccessorDeclaration, getAnnotatedType: (a: AccessorDeclaration) => Type | undefined, message: DiagnosticMessage) {
+        function checkAccessorDeclarationTypesAssignable(getter: AccessorDeclaration, setter: AccessorDeclaration, getAnnotatedType: (a: AccessorDeclaration) => Type | undefined, message: DiagnosticMessage) {
+            return checkAccessorDeclarationTypesMatch(getter, setter, getAnnotatedType, isTypeAssignableTo, message);
+        }
+
+        function checkAccessorDeclarationTypesMatch(first: AccessorDeclaration, second: AccessorDeclaration, getAnnotatedType: (a: AccessorDeclaration) => Type | undefined, match: typeof areTypesComparable, message: DiagnosticMessage) {

```

Comment:
Is this used outside of the assignability check immediately above?

Reply:
there used to be a useless error that got removed in the course of this PR (the call was on line 32991)

Reply:
Not anymore. I can inline it